<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGoLineageViewerWidget</class>
 <widget class="QDockWidget" name="QGoLineageViewerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>561</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::AllDockWidgetFeatures</set>
  </property>
  <property name="windowTitle">
   <string>Lineage Viewer</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QDockWidget" name="tableDockWidget">
      <property name="floating">
       <bool>false</bool>
      </property>
      <property name="features">
       <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
      </property>
      <property name="allowedAreas">
       <set>Qt::AllDockWidgetAreas</set>
      </property>
      <property name="windowTitle">
       <string>Table</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QFrame" name="tableFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDockWidget" name="graphDockWidget">
      <property name="floating">
       <bool>false</bool>
      </property>
      <property name="features">
       <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
      </property>
      <property name="allowedAreas">
       <set>Qt::AllDockWidgetAreas</set>
      </property>
      <property name="windowTitle">
       <string>Graph</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_3">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QFrame" name="graphFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QVTKWidget" name="graphViewWidget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>1</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>150</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDockWidget" name="controlDockWidget">
      <property name="floating">
       <bool>false</bool>
      </property>
      <property name="features">
       <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
      </property>
      <property name="allowedAreas">
       <set>Qt::AllDockWidgetAreas</set>
      </property>
      <property name="windowTitle">
       <string>Paramters</string>
      </property>
      <widget class="QWidget" name="dockWidgetContents_4">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_15">
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <widget class="QCheckBox" name="radialCheckBox">
               <property name="text">
                <string>Radial Layout</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="0">
                <widget class="QSlider" name="radialSlider">
                 <property name="maximum">
                  <number>360</number>
                 </property>
                 <property name="value">
                  <number>360</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="radialSpinBox">
                 <property name="maximum">
                  <number>360</number>
                 </property>
                 <property name="value">
                  <number>360</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QGridLayout" name="gridLayout_12">
             <item row="0" column="0">
              <widget class="QCheckBox" name="logCheckBox">
               <property name="text">
                <string>Log</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="logSpinBox">
               <property name="minimum">
                <double>0.100000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="0" column="0">
              <widget class="QCheckBox" name="colorCheckBox">
               <property name="text">
                <string>Color Code</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="colorComboBox"/>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="0" column="0">
              <widget class="QCheckBox" name="scaleCheckBox">
               <property name="text">
                <string>Scale</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="scaleComboBox"/>
             </item>
            </layout>
           </item>
           <item row="5" column="0">
            <layout class="QGridLayout" name="gridLayout_11">
             <item row="0" column="0">
              <widget class="QCheckBox" name="labelCheckBox">
               <property name="text">
                <string>Label</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="labelComboBox"/>
             </item>
            </layout>
           </item>
           <item row="6" column="0">
            <widget class="QCheckBox" name="backCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Back Plane</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <layout class="QGridLayout" name="gridLayout_13">
             <item row="0" column="0">
              <widget class="QPushButton" name="addLineagePushButton">
               <property name="text">
                <string>Add Lineage</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="deleteLineagePushButton">
               <property name="text">
                <string>Delete Lineage</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
